Cloud Application Manager/Getting Started/getting-started-login-quick-tour.md

```diff
@@ -18,21 +18,21 @@
 * [Quick tour](#quick-tour)
 
  ### Overview
- This article makes a quick tour on login page and the panel after login to the platform.
+ This article makes a quick tour on login page and the panel after login to the platform in Applications site.
```

Comment:
Hey @julio-castanar, this article is completely outdated, so please review it fully. It would require more changes, just follow the article and change the comments that do not correspond to the actual behavior. Thanks!